src/event/event.php

```diff
@@ -186,12 +194,117 @@ private static function pixChargebackLogResource()
+
+    ## Return:
+        - Event object with updated attributes
+     */
+    public static function get($id, $user = null)
+    {
+        return Rest::getId($user, Event::resource(), $id);
+    }
+
+    /**
+    # Retrieve notification Events
+
+    Receive a enumerator of notification Event objects previously created in the Stark Infra API
+
+    ## Parameters (optional):
+        - limit [integer, default null]: maximum number of objects to be retrieved. Unlimited if null. ex: 35
+        - after [DateTime or string, default null]: date filter for objects created only after specified date. ex: "2020-04-03"
+        - before [DateTime or string, default null]: date filter for objects created only before specified date. ex: "2020-04-03"
+        - isDelivered [bool, default null]: bool to filter successfully delivered events. ex: true or false
+        - user [Organization/Project object, default null]: Organization or Project object. Not necessary if StarkInfra\Settings::setUser() was used before function call
+
+    ## Return:
+        - enumerator of Event objects with updated attributes
+     */
+    public static function query($options = [], $user = null)
+    {
+        
+        $options["after"] = new StarkDate(Checks::checkParam($options, "after"));
+        $options["before"] = new StarkDate(Checks::checkParam($options, "before"));
+        return Rest::getList($user, Event::resource(), $options);
+    }
+
+    /**
+    # Retrieve paged Events
+
+    Receive a list of up to 100 Event objects previously created in the Stark Infra API and the cursor to the next page.
+    Use this function instead of query if you want to manually page your requests.
+
+    ## Parameters (optional):
+    - cursor [string, default null]: cursor returned on the previous page function call
+    - limit [integer, default 100]: maximum number of objects to be retrieved. It must be an integer between 1 and 100. ex: 50
+    - after [DateTime or string, default null] date filter for objects created only after specified date. ex: "2020-04-03"
+    - before [DateTime or string, default null] date filter for objects created only before specified date. ex: "2020-04-03"
+    - isDelivered [boolean, default None]: bool to filter successfully delivered events. ex: True or False
+    - user [Organization/Project object, default null, default null]: Organization or Project object. Not necessary if StarkInfra\Settings::setUser() was set before function call
+    
+    ## Return:
+    - list of Event objects with updated attributes
+    - cursor to retrieve the next page of Event objects
+     */
+    public static function page($options = [], $user = null)
+    {
+        return Rest::getPage($user, Event::resource(), $options);
+    }
+
+    /**
+    # Delete notification Events
+
+    Delete an array of notification Event entities previously created in the Stark Infra API
```

Comment:
tocar para:

Delete a single Event entity previously created in the Stark Infra API